src/map/map.c

```diff
@@ -449,6 +449,8 @@ static int map_count_oncell(int16 m, int16 x, int16 y, int type, int flag)
 	struct block_list *bl;
 	int count = 0;
 
+	Assert_ret(m < 0 || m >= map->count);
```

Comment:
look like all assert checks is wrong.
condition in assert should be for correct values. but here look like for invalid values.

Reply:
yeah sorry, I noticed after pushing...

I guess I shouldn't work while tired haha